client/src/components/DetailView.tsx

```diff
@@ -414,25 +397,24 @@ class DetailViewWithoutI18n extends Component<Props, State> {
                     </div>
                   </div>
                   <div className="column col-lg-12 col-5">
-                    <div className="card-image hide-lg">{streetView}</div>
-                    <div className="card-body column-right">
+                    <EmailAlertSignup
+                      bbl={detailAddr.bbl}
+                      housenumber={detailAddr.housenumber}
+                      streetname={detailAddr.streetname}
+                      zip={detailAddr.zip || ""}
+                      boro={detailAddr.boro}
```

Comment:
it really doesn't matter, but if we wanted I think both `<EmailAlertSignup>` and `<BuildingSubscribe>` could just take `props: { addr: AddressRecord }`

Reply:
that's a great catch. will make that change!